core/src/test/java/com/google/cloud/sql/core/CoreSocketFactoryTest.java

```diff
@@ -179,12 +177,13 @@ public void create_throwsErrorForInvalidInstanceRegion() throws IOException {
                 .setRegion("beer"));
 
     CoreSocketFactory coreSocketFactory =
-        new CoreSocketFactory(new Clock(), clientKeyPair, credential, adminApi, 3307);
+        new CoreSocketFactory(clientKeyPair, credential, adminApi, 3307);
     try {
       coreSocketFactory.createSslSocket("foo:bar:baz", Arrays.asList("PRIMARY"));
       fail();
     } catch (IllegalArgumentException e) {
-      assertThat(e.getMessage()).startsWith("Incorrect region value");
+      assertThat(e.getMessage())
```

Comment:
```suggestion
      assertThat(e).hasMessageThat()
```

Do this throughout, and you'll get better failure messages.